torchmd/run.py

```diff
@@ -124,6 +125,9 @@ def get_args(arguments=None):
         type=tuple,
         help="exclusions for the LJ or repulsionCG term",
     )
+    parser.add_argument(
+        "--max_interatomic_dist", type=float, default=0.5, help="Maximum interatomic distance as stability parameter"
+    )
```

Comment:
0.5 sounds like a really arbitrary number. If anything, I would put something like -1 to be understood as "do nothing". You normally try to add functionality in a retrocompatible way.